docs/getting-started.md

```diff
@@ -311,6 +311,43 @@ Content-Length: 0
 ```
 
 ![New computation](img/ui/new_computation.png)
+Before running a computation, user roles in the computation need to be assigned and the all the required assets created and linked to the computation.
+
+### Assigning Computation Roles and Permissions
+
+1. Navigate to the roles page from the computation details page.
+![Roles](img/ui/roles.png)
+
+2. Select a role to which you would like to add a user.
+Please note that this user needs to be invited to a workspace and to have accepted the invitation.
+![Roles](img/ui/view_role.png)
+
+3. Switch to the members tab and click on the Add Members button.
+![Roles](img/ui/role_details.png)
+
+4. Search for the user you'd like to assign a role.
+
+5. Select the user from the list and click on Add Selected Members button.
+![Roles](img/ui/add_user_to_role.png)
+
+6. Upon successful role assignment, the user will appear on the role details page.
+![Roles](img/ui/assigned_user_role.png)
+
```

Comment:
we need to be specific with the roles required for a computation which at minimum is an algo provider and result consumer all which require a user